backtester/README.md

```diff
@@ -0,0 +1,106 @@
+[![Go Report Card](https://goreportcard.com/badge/github.com/thrasher-corp/gocryptotrader)](https://goreportcard.com/report/github.com/thrasher-corp/gocryptotrader)
+
+
+This backtester package is part of the GoCryptoTrader codebase.
+
+## This is still in active development
+
+You can track ideas, planned features and what's in progress on this Trello board: [https://trello.com/b/ZAhMhpOy/gocryptotrader](https://trello.com/b/ZAhMhpOy/gocryptotrader).
+
+Join our slack to discuss all things related to GoCryptoTrader! [GoCryptoTrader Slack](https://join.slack.com/t/gocryptotrader/shared_invite/enQtNTQ5NDAxMjA2Mjc5LTc5ZDE1ZTNiOGM3ZGMyMmY1NTAxYWZhODE0MWM5N2JlZDk1NDU0YTViYzk4NTk3OTRiMDQzNGQ1YTc4YmRlMTk)
+
+
+# GoCryptoTrader Backtester
+An event-driven backtesting tool to test and iterate trading strategies using historical or custom data.
+
+## Features
+- Works with all GoCryptoTrader exchanges that support trade/candle retrieval. See [candle readme](/docs/OHLCV.md) and [trade readme](/exchanges/trade/README.md) for supported exchanges
+- CSV data import
+- Database data import
+- Proof of concept live data running
+- Can run strategies against multiple cryptocurrencies
+- Can run strategies that can assess multiple currencies simultaneously to make complex decisions
+- Dollar cost strategy implementation
+- RSI strategy implementation
+- Rules customisation via config `.strat` files
+- Strategy customisation without requiring recompilation. For example, customising RSI high, low and length values via config `.strat` files.
+- Report generation
+- Portfolio manager to help size orders based on config rules, risk and candle volume
+- Order manager to place orders with customisable slippage estimator
+- Helpful statistics to help determine whether a strategy was effective
+- Compliance manager to keep snapshots of every transaction and their changes at every interval
+
+## How does it work?
+- The application will load a `.strat` config file as specified at runtime
+- The `.strat` config file will contain
+  - Start & end dates
+  - The strategy to run
+  - The candle interval
+  - Where the data is to be sourced ([API](/backtester/data/kline/api), [CSV](/backtester/data/kline/csv), [database](/backtester/data/kline/database), [live](/backtester/data/kline/live))
+  - Whether to use trade or candle data ([readme](/data/kline/README.md))
+  - A nickname for the strategy (to help differentiate between runs/configs using the same strategy)
+  - The currency/currencies to use
+  - The exchange(s) to run against
+  - See [readme](/config/README.md) for a breakdown of all config features
+- The GoCryptoTrader Backtester will retrieve the data specified in the config ([readme](/backtest/README.md))
+- The data is converted into candles and each candle is streamed as a data event.
+- The data event is analysed by the strategy which will output a purchasing signal such as `BUY`, `SELL` or `DONOTHING` ([readme](/eventtypes/signal))
```

Comment:
```suggestion
- The data event is analysed by the strategy which will output a purchasing signal such as `BUY`, `SELL` or `DONOTHING` ([readme](/backtester/eventtypes/signal/README.md))
```

Will stop reporting on these, but there's quite a lot of relative deadlinks throughout the various readme's